.circleci/PR_SITE_NUM.sh

```diff
@@ -0,0 +1,3 @@
+PR_NUM="$(echo "$CIRCLE_PULL_REQUEST" | perl -ne "/(\d+)\$/; print \$1")"
```

Comment:
Shebang lines in shell scripts are to ensure compatibility. This line should work in any shell, including `sh`, so it would be inappropriate to limit to a particular shell using a shebang line.